Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.1
```

Comment:
Bumping this version breaks CI testing of the package, as we're currently testing it with Xcode 10.